lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -663,6 +664,9 @@ func convertContainerToContainerInfo(c *exec.Container) *models.ContainerInfo {
 			ep.Ports = append(ep.Ports, endpoint.Ports...)
 		}
 
+		macaddress, _ := container.GetContainerVM().GetMacAddressBasedSpecifiedNetworkName(context.Background(), endpoint.Assigned.IP.String())
```

Comment:
It seems like we should store the returned `error` and log it if it's not `nil`.

Reply:
Ok, I will log the error msg. But we will set macaddress to "" if the error is returned, I do not think the docker inspect will be blocked due to macaddress.

Reply:
I agree that blocking doesn't seem appropriate.